src/conky.cc

```diff
@@ -1961,6 +1962,40 @@ void load_config_file() {
   lua::stack_sentry s(l);
   l.checkstack(2);
 
+  // Extend lua package.path so scripts can use relative paths
+  {
+    struct stat file_stat {};
+
+    std::string path_ext;
+
+    // add XDG directory to lua path
+    auto xdg_path =
+        std::filesystem::path(to_real_path(XDG_CONFIG_FILE)).parent_path();
+    if (stat(xdg_path.c_str(), &file_stat) == 0) {
+      path_ext.push_back(';');
+      path_ext.append(xdg_path);
+      path_ext.append("/?.lua");
+    }
+
+    auto parent_path = current_config.parent_path();
+    if (xdg_path != parent_path && stat(path_ext.c_str(), &file_stat) == 0) {
+      path_ext.push_back(';');
+      path_ext.append(parent_path);
+      path_ext.append("/?.lua");
+    }
+
+    l.getglobal("package");
+    l.getfield(-1, "path");
+
+    auto path = l.tostring(-1);
+    path.append(path_ext);
+    l.pop();
+    l.pushstring(path.c_str());
+
+    l.setfield(-2, "path");
+    l.pop();
+  }
```

Comment:
Adding `$HOME/.config/conky` and config parent directory to lua path before loading the script.